components/SearchBar/SearchBar.js

```diff
@@ -102,6 +105,8 @@ const StyledSearchResults = styled.div`
   overflow-y: auto;
   border-radius: 10px;
   background-color: var(--color-beige);
+  z-index: 2;
+  border: 2px green solid;
 
```

Comment:
I would style the searchbar a little different because if you're on the Kirschblütental-List for example, the search bar is very hardly visible. Maybe a background color would be good :)

Reply:
Done!